projects/cairographics.org/pycairo/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/pygobject/pycairo/releases/download/v{{version}}/pycairo-{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: pygobject/pycairo
+dependencies:
+  cairographics.org: '*'
+runtime:
+  env:
+    PYTHONPATH: "{{prefix}}/lib/python{{deps.python.org.version.marketing}}/site-packages:$PYTHONPATH"
```

Comment:
Actually, this probably won't work. Since it doesn't have a runtime dep on python. So, we'd either have to hardcode it to 3 (which is probably what we need to do), or something else that probably won't work.

Reply:
Yes, you are right. I did not think about the fact that Python is not in runtime